YUViewLib/src/ffmpeg/AVFormatContextWrapper.cpp

```diff
@@ -180,6 +180,42 @@ typedef struct AVFormatContext_59
   AVDictionary *         metadata;
 } AVFormatContext_59;
 
+typedef struct AVFormatContext_60
+{
+  const AVClass *        av_class;
+  struct AVInputFormat * iformat;
+  struct AVOutputFormat *oformat;
+  void *                 priv_data;
+  AVIOContext *          pb;
+  int                    ctx_flags;
+  unsigned int           nb_streams;
+  AVStream **            streams;
+  char *                 url;
+  int64_t                start_time;
+  int64_t                duration;
+  int64_t                bit_rate;
+  unsigned int           packet_size;
+  int                    max_delay;
+  int                    flags;
+  int64_t                probesize;
+  int64_t                max_analyze_duration;
+  const uint8_t *        key;
+  int                    keylen;
+  unsigned int           nb_programs;
+  AVProgram **           programs;
+  enum AVCodecID         video_codec_id;
+  enum AVCodecID         audio_codec_id;
+  enum AVCodecID         subtitle_codec_id;
+  unsigned int           max_index_size;
+  unsigned int           max_picture_buffer;
+  unsigned int           nb_chapters;
+  AVChapter **           chapters;
+  AVDictionary *         metadata;
+  int64_t                start_time_realtime;
+  int                    fps_probe_size;
+  int                    error_recognition;
```

Comment:
Why add these 3 values to the struct? They are not used anywhere in the code ...